### ISSUE TITLE
Review design assets

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/Mozfest2016_production#11]
Review the list of [design assets needed for 2016] (https://docs.google.com/spreadsheets/d/1J4LS4ILtG5x-jxwHEwGuDzXHRHC9IQM4t9fg2OUeRBs/edit#gid=1773878800)

Identify which should be done in-house and which can be done by a contractor.

**Comment by edrushka** [comment]
@Saallen @xmatthewx  I've taken a first pass at identifying which tasks should be done by a MoFo designer, and which can be done by the contractor.

See column D in the [2016 Design Assets sheet] (https://docs.google.com/spreadsheets/d/1J4LS4ILtG5x-jxwHEwGuDzXHRHC9IQM4t9fg2OUeRBs/edit#gid=1773878800)

**Comment by kristinashu** [comment]
I think if we found the right contractor I would be happy to even let them determine the overall style of MozFest (with lots of guidance and feedback from us). If we found a fresh bright-eyed grad they could create something very cool. Some side benefits might be that by starting a contractor earlier on in the process they would be more invested, attached and motivated by the design direction. It would give us a chance to test and build our relationship with the contractor sooner rather than later (giving us time to find a replacement if they fall through). It also just seems to make sense given our limited design resources at the moment. Let me know what you think!

**Comment by xmatthewx** [comment]
I'm open to it, but not ready to make a plan that depends on it. Managing a contractor can take as much time as doing the work yourself if you don't have the right person and tasks aligned. A contractor doing production work is less risky and a bigger time saver for us. 

To make it possible, we'll have to ramp up the search process.

**Comment by edrushka** [comment]
I'm going to mark this as closed since I feel we've covered this both on the assets spreadsheet and in the Junior Designer's scope of work #12.

To summarize: Sabrina will develop the 2016 Style Guide and key graphic assets. The Junior designer will assist with more of the production-based work.

It's worth re-stating for the record that although we've listed design and installation of wayfinding assets and last-minute speaker slide changes as potential tasks (we listed both to have maximum flexibility), it's unrealistic to assume the junior designer can do *both*, since the timing overlaps. Mofo designers will have to help here (most likely on the speaker slides, since that's working directly with Mark S).
